app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
we already have those colors defined, they are under legacy/ folder instead of main/ folder

Reply:
removed.